version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "0.9.3-SNAPSHOT"
+ThisBuild / version := "0.10.1-SNAPSHOT"
```

Comment:
Not sure why this jumped from 0.9.3 to 0.10.1, where was 0.10.0 published from?

Reply:
it was published from the branch `0.10.x`, there are several commits that do the transition:
![image](https://user-images.githubusercontent.com/1832751/143257393-90d35929-b2e6-432c-95fd-d0e8bf9dc887.png)

log:
![image](https://user-images.githubusercontent.com/1832751/143257878-5e8eb6c1-52ab-45eb-b005-f96b2c2b1dfd.png)

Opted to have a new branch `0.10.x` instead of continuing `0.9.x` because the amount of updated libraries was too big. In the worst case it might introduce compatibly issues and run-time errors (as it was before) in projects that use the library.

Reply:
@javierarrieta 
any objections to merge it?

Reply:
Sorry, missed this, no, not at all